CONTRIBUTING.md

```diff
@@ -94,8 +94,8 @@ auto:
   # Use "library" as the owner name for a official docker/community image
   - dockerhub: ministryofmagic/timeturner
 
-  # Link to package on NPM
-  - npm: https://www.npmjs.com/package/abc
+  # Identifier of the package on NPM
```

Comment:
@marcwrobel could we remove this part from this pr and fix it in #2061

Reply:
Sure, one less conflict during rebase ;). Will do it now.

Reply:
Done.